specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -4026,10 +4026,34 @@
         "forwardDeadLetteredMessagesTo": {
           "type": "string",
           "description": "Queue/Topic name to forward the Dead Letter message"
+        },
+        "isClientAffine": {
```

Comment:
new writable properties are a breaking change that should go in a new api-version.  Older clients for this api-version won't provide this which should revert to default values which most likely is not what the user wants.  See "breaking changes" [here](https://armwiki.azurewebsites.net/api_contracts/Intro.html)

Reply:
we will add this to the new api version.